src/api/app/assets/stylesheets/webui/texts.scss

```diff
@@ -18,6 +18,10 @@
   word-break: break-all;
 }
 
+.text-word-break-word {
```

Comment:
I would use https://getbootstrap.com/docs/4.5/utilities/text/#word-break, instead of adding a custom class.